docs/Documentation/Drivers/Classic/DefaultOutputs.md

```diff
@@ -4,30 +4,30 @@ If the user does not give explicit output file instructions in the global parame
 
 ## Flux Files
 
-The primary output file type for the VIC-NL model is the flux file, which contains information about moisture and energy fluxes for each time step. These output files are based on the model output files used for the PILPS-2C project. The number of variables in this file depends on the values of FULL_ENERGY and FROZEN_SOIL in the [global parameter file](GlobalParam.md); when either FULL_ENERGY or FROZEN_SOIL are true, the file will contain the same number of columns as found in the PILPS-2C files for comparison purposes. Flux files are always written, regardless of the mode of operation. Flux files begin with the prefix "fluxes_".
+The primary output file type for the VIC model is the flux file, which contains information about moisture and energy fluxes for each time step. These output files are based on the model output files used for the PILPS-2C project. The number of variables in this file depends on the values of `FULL_ENERGY` and `FROZEN_SOIL` in the [global parameter file](GlobalParam.md); when either `FULL_ENERGY` or `FROZEN_SOIL` are true. Flux files are always written, regardless of the mode of operation. Flux files begin with the prefix `fluxes_`.
 
```

Comment:
Should be:  `The number of variables in this file depends on the values of `FULL_ENERGY` and `FROZEN_SOIL` in the [global parameter file](GlobalParam.md).  When either `FULL_ENERGY` or `FROZEN_SOIL` are true, flux files are always written, regardless of the mode of operation. Flux files begin with the prefix `fluxes_`.`